src/rules_exceptions.cc

```diff
@@ -212,7 +212,7 @@ bool RulesExceptions::merge(RulesExceptions *from) {
             return ret;
         }
     }
-    for (auto b : from->m_ranges) {
+    for (auto& b : from->m_ranges) {
```

Comment:
```suggestion
    for (const auto& b : from->m_ranges) {
```
Please see cppcheck's [warning](https://github.com/owasp-modsecurity/ModSecurity/actions/runs/19540276744/job/55952868889?pr=3468#step:5:3290):
```
warning: src/rules_exceptions.cc,215,style,constVariableReference,Variable 'b' can be declared as reference to const
```